Gruntfile.js

```diff
@@ -238,33 +46,36 @@ module.exports = function(grunt) {
           port: 9696
         }
       }
+    },
+
+    run: {
+      requireJquery: {
```

Comment:
Let's add some comments to describe what we're building with each of these two targets (i.e. I had to look at the config files to figure it out).

Also, better names - i.e. requireJquery doesn't tell me that this is in fact "the version that does not include jquery or q as part of the minified file".